.github/CODEOWNERS

```diff
@@ -6,3 +6,8 @@
 include/swift/Parse @ahoppen @bnbarham @CodaFi @DougGregor @rintaro
 lib/Parse           @ahoppen @bnbarham @CodaFi @DougGregor @rintaro
 test/Parse          @ahoppen @bnbarham @CodaFi @DougGregor @rintaro
+
+# C++ Interop
```

Comment:
@zoecarver @hyp do you think we should also auto-add ourselves as reviewers for all of ClangImporter? We could also try to filter the source files within ClangImporter that are related to C++ interop if that makes sense.

Reply:
All of ClangImporter/PrintAsClang might make sense, but I think that warrants a wider discussion with the Swift team, they might want to add an additional code owner there as well.